stdlib/mime/src/main/java/org/ballerinalang/mime/nativeimpl/AbstractGetPayloadHandler.java

```diff
@@ -0,0 +1,117 @@
+/*
+*  Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing,
+*  software distributed under the License is distributed on an
+*  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+*  KIND, either express or implied.  See the License for the
+*  specific language governing permissions and limitations
+*  under the License.
+*/
+
+package org.ballerinalang.mime.nativeimpl;
+
+import io.netty.handler.codec.http.HttpHeaderNames;
+import org.ballerinalang.bre.Context;
+import org.ballerinalang.bre.bvm.CallableUnitCallback;
+import org.ballerinalang.mime.util.EntityBodyHandler;
+import org.ballerinalang.mime.util.HeaderUtil;
+import org.ballerinalang.mime.util.MimeUtil;
+import org.ballerinalang.model.NativeCallableUnit;
+import org.ballerinalang.model.util.StringUtils;
+import org.ballerinalang.model.values.BError;
+import org.ballerinalang.model.values.BMap;
+import org.ballerinalang.model.values.BString;
+import org.ballerinalang.model.values.BValue;
+import org.ballerinalang.stdlib.io.utils.BallerinaIOException;
+import org.wso2.transport.http.netty.message.FullHttpMessageListener;
+import org.wso2.transport.http.netty.message.HttpCarbonMessage;
+import org.wso2.transport.http.netty.message.HttpMessageDataStreamer;
+
+import static org.ballerinalang.mime.util.MimeConstants.CHARSET;
+import static org.ballerinalang.mime.util.MimeConstants.ENTITY_BYTE_CHANNEL;
+import static org.ballerinalang.mime.util.MimeConstants.TRANSPORT_MESSAGE;
+import static org.ballerinalang.mime.util.MimeUtil.isNotNullAndEmpty;
+
+/**
+ * {@code AbstractGetPayloadHandler} is the base class for all get entity body functions.
+ *
+ * @since 0.990.4
```

Comment:
As per our new release stratergy next version should be 0.995-r1
```suggestion
 * @since 0.995-r1
```